ext-src/BrowserViewWindow.ts

```diff
@@ -162,6 +162,12 @@ export class BrowserViewWindow extends EventEmitter.EventEmitter2 {
     });
   }
 
+  public show() {
```

Comment:
This is used so that the user can select a browser in the shared browsers list, and have it automatically reveal the associated web view panel.